iocage_lib/ioc_start.py

```diff
@@ -857,7 +892,7 @@ def start_copy_localtime(self):
 
     def start_generate_resolv(self):
         resolver = self.get("resolver")
-        #                                     compat
+        # compat
```

Comment:
That whitespace was meant to directly reference the `resolver != none` bit below it :) So it served a purpose.